e2e/ci-e2e/mocks/setup.ts

```diff
@@ -28,6 +30,7 @@ export async function setupTestRepo(folder: string) {
   );
 
   await cp(fixturesDir, baseDir, { recursive: true });
+  await restoreRenamedFiles(baseDir, NX_IGNORED_FILES_TO_RESTORE);
```

Comment:
I see the second argument is always set to `NX_IGNORED_FILES_TO_RESTORE` constant in each usage. How about making it the default value? Or hard-coding it even? It would make usages less verbose.

Reply:
Will hardcode it.